insights/client/client.py

```diff
@@ -88,79 +87,88 @@ def set_up_logging(config):
         logger.debug("Logging initialized")
 
 
-def test_connection(config):
+def get_connection(config):
+    return InsightsConnection(config)
+
+
+def test_connection(pconn):
     """
     Test the connection
     """
-    pconn = get_connection(config)
     return pconn.test_connection()
 
 
-def _is_client_registered(config):
```

Comment:
Did you change the order of the functions?  This diff is so hard to read.

Reply:
yeah my bad. lemme put them back in the old order so it's easier to parse

Reply:
Ah! So much better! ❤️